### PR TITLE
:wrench: Update create_default_site command

### LINKED ISSUE #836: Posting comment redirects to example.com
For instance, here: https://global.iogt.site/fr/sections/emergency-information/about-covid-19/about-coronavirus/

Site object is configured as global.iogt.site port 80. Also tried global.iogt.site:443, http://global.iogt.site:80, https://global.iogt.site:443, same result regardless.

### PR DESCRIPTION
**Pull request by ehmadzubair**
Closes #836

### CODE REVIEW

**Comment by geoo89** [comment]
Could you please make this command update both the Site object in django (which is currently does) but also the wagtail Site object which we see in the CMS? Thanks!

**Comment by sheralim012** [comment]
@geoo89 done

File: home/management/commands/create_default_site.py [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+        domain = options.get('domain') or 'localhost:8000'
+
+        __, created = DjangoSite.objects.update_or_create(pk=settings.SITE_ID, defaults={
+            'domain': domain,
             'name': 'default'
         })
-        self.stdout.write(self.style.SUCCESS(f'Site object {"created" if created else "updated"}'))
+        self.stdout.write(self.style.SUCCESS(f'Django Site object {"created" if created else "updated"}'))
+        __, created = WagtailSite.objects.update_or_create(is_default_site=True, defaults={
+            'hostname': domain,
```
Comment (geoo89) [inline home/management/commands/create_default_site.py:29]: Only the hostname (and port) should be updated, not the other fields of the site object.

File: home/management/commands/create_default_site.py [hunk 2]
```diff
@@ -1,16 +1,33 @@
...
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--domain',
+            help='The domain on which the website is deployed'
+        )
+
     def handle(self, *args, **options):
-        _, created = Site.objects.update_or_create(pk=settings.SITE_ID, defaults={
-            'domain': 'localhost:8000',
+        domain = options.get('domain') or 'localhost:8000'
```
Comment (geoo89) [inline home/management/commands/create_default_site.py:21]: It appears that the DjangoSite domain does not contain a port (at least, if I specify a port, I get an error when posting a comment, see #836, while it works ok if I only specify the URL), so the default should probably be simply `localhost`?

File: home/management/commands/create_default_site.py [hunk 3]
```diff
@@ -1,16 +1,33 @@
...
 User = get_user_model()
 
 
 class Command(BaseCommand):
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--domain',
+            help='The domain on which the website is deployed'
+        )
```
Comment (geoo89) [inline home/management/commands/create_default_site.py:18]: Can we add a port parameter which affects the WagtailSite object? (It seems that the DjangoSite object does not have a port associated with it)

**Review by geoo89 — COMMENTED** [review]
A few issues to address here.

**Comment by sheralim012** [comment]
@geoo89 added port argument for wagtail site object and simple domain argument will be used for django domain/hostname

**Pull request merged**